src/plopp/backends/matplotlib/image.py

```diff
@@ -175,3 +202,37 @@ def update(self, new_values: sc.DataArray):
             New data to update the mesh values from.
         """
         self._data = new_values
+
+    def format_coord(self, x: float, y: float) -> str:
+        """
+        Format the coordinates of the mouse pointer to show the value of the
+        data at that point.
+
+        Parameters
+        ----------
+        x:
+            The x coordinate of the mouse pointer.
+        y:
+            The y coordinate of the mouse pointer.
+        """
+        out = f'x={x:1.4f}, y={y:1.4f}'
+        xy = {'x': x, 'y': y}
+        try:
+            val = self._data_with_bin_edges[
+                self._hover_slicing['dim'][0],
+                sc.scalar(
+                    xy[self._hover_slicing['dir'][0]],
+                    unit=self._hover_slicing['unit'][0],
+                ),
+            ]
+            val = val[
+                self._hover_slicing['dim'][1],
+                sc.scalar(
+                    xy[self._hover_slicing['dir'][1]],
+                    unit=self._hover_slicing['unit'][1],
+                ),
+            ]
+            out += f', z={val.value:1.4f}'
```

Comment:
Same question, do we need different formatting, e.g., scientific?

Further, should we include the units, I think it would improve the UX?

Can we make use of the "compact" variable formatting?

Reply:
Good point about the unit. Do we also want to add the units for the cursor values in 1d plots?

Reply:
Yes.